### PR TITLE
config option not work issue

### PR DESCRIPTION
**Pull request by kazupon**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -54,19 +56,25 @@ var writerOpts;
 
 try {
   if (flags.context) {
-    templateContext = require(flags.context);
+    templateContext = require(resolve(process.cwd(), flags.context));
```
Comment (stevemao) [inline cli.js]: I don't think this is needed

**Comment by stevemao** [comment]
Can you add a test?

**Comment by stevemao** [comment]
`This value is ignored if preset is specified',` should be removed. Ref: https://github.com/conventional-changelog/conventional-changelog-cli/commit/a0449a237cad384a8bd601593e5b538ae0bdfa56

**Comment by kazupon** [comment]
I added a test.
Please check.

**Comment by stevemao** [comment]
I'm giving you push access @kazupon

**Comment by kazupon** [comment]
Thanks!! @stevemao
Do you mean that I may push the this repsitory directly ? or merge only ?

**Comment by stevemao** [comment]
up to you @kazupon :)

**Comment by kazupon** [comment]
Thanks!! @stevemao 

I'm interested in conventional-changelog project.
the below the my project:
https://github.com/kazupon/git-commit-message-convention

I think that people are become happy in software development!

Please join me if that is possible :)

**Comment by stevemao** [comment]
Added you to the org. 
I think the biggest issue is https://github.com/conventional-changelog/conventional-changelog-writer/issues/20. I'm not exactly sure about how the API should be.

Similar to your project, please have a look at https://github.com/conventional-changelog/conventional-changelog/pull/146, Thanks for the contribution :)

**Comment by kazupon** [comment]
Thanks!
At first, I understand the this project, and I'll try to challenge it. :)

**Comment by stevemao** [comment]
@kazupon tell me your npm user name if you want access :)

**Comment by kazupon** [comment]
@stevemao 
Thanks!
my npm user name is `kazupon`. :)

**Comment by stevemao** [comment]
@kazupon Done :)

**Pull request merged**